package.nls.json

```diff
@@ -0,0 +1,96 @@
+    "cmake-tools.configuration.cmake.parallelJobs.description": "The number of parallel build jobs. Use zero to automatically detect the number of CPUs.",
+    "cmake-tools.configuration.cmake.ctestPath.description": "Path to CTest executable. If null, will be inference from cmake.cmakePath (recommended to leave null)",
+    "cmake-tools.configuration.cmake.ctest.parallelJobs.description": "The number of parallel test jobs. Use zero to use the value of cmake.parallelJobs",
+    "cmake-tools.configuration.cmake.parseBuildDiagnostics.description": "Parse compiler output for warnings and errors",
+    "cmake-tools.configuration.cmake.enabledOutputParsers.description": "Output parsers to use. Supported parsers 'cmake', 'gcc', 'gnuld' for GNULD-style inker output, 'msvc' for Microsoft Visual C++, and 'ghs' for the Green Hills compiler with --no_wrap_diagnostics --brief_diagnostics",
+    "cmake-tools.configuration.cmake.debugConfig.description": "The debug configuration to use when debugging a target",
+    "cmake-tools.configuration.cmake.debugConfig.symbolSearchPath.description": "Visual Studio debugger symbol search paths",
+    "cmake-tools.configuration.cmake.debugConfig.additionalSOLibSearchPath.description": "Paths for GDB or LLDB to search for .so files",
+    "cmake-tools.configuration.cmake.debugConfig.externalConsole.description": "Launch an external console for the program",
+    "cmake-tools.configuration.cmake.debugConfig.logging.description": "Tell what types of messages should be logged to the console",
+    "cmake-tools.configuration.cmake.debugConfig.visualizerFile.description": ".natvis file to be used when debugging",
+    "cmake-tools.configuration.cmake.debugConfig.args.description": "Arguments to pass to program command line",
+    "cmake-tools.configuration.cmake.debugConfig.cwd.description": "Set the working directory for the program",
+    "cmake-tools.configuration.cmake.debugConfig.environment.name.description": "Name of environment variable",
+    "cmake-tools.configuration.cmake.debugConfig.environment.value.description": "Value for the environment variable",
+    "cmake-tools.configuration.cmake.debugConfig.miDebuggerPath.description": "Path to gdb or lldb debugger",
+    "cmake-tools.configuration.cmake.debugConfig.stopAtEntry.description": "Stop at the entry point of the target",
+    "cmake-tools.configuration.cmake.debugConfig.setupCommands.description": "Command to set up gdb or lldb",
+    "cmake-tools.configuration.cmake.debugConfig.setupCommands.text.description": "Command to run",
+    "cmake-tools.configuration.cmake.debugConfig.setupCommands.description.description": "Description of the command",
+    "cmake-tools.configuration.cmake.defaultVariants.buildType.description": "The build type",
+    "cmake-tools.configuration.cmake.defaultVariants.buildType.debug.long": "Emit debug information without performing optimizations",
```

Comment:
Let's make the descriptions for these align with each other (pick a separator that looks good 😄)

Debug -> "Disable optimizations - include debug information"
Release -> "Optimize for speed - exclude debug information"
MinSizeRel -> "Optimize for smallest binary size - exclude debug information"
RelWithDebInfo -> "Optimize for speed - include debug information"